instructions/section-1/lesson-1-4.md

```diff
@@ -0,0 +1,130 @@
+[< Back to Overview](../../README.md)
+
+# Lesson 1.4: Advanced React Props and State
+
+This assignment will teach you the following:
+
+- Lifting State
+- Controlled Components
+- Props handling
+
+---
+
+## Instructions
+
+### Getting Started:
+
+Merge your pull request from the previous lesson (if you haven't already):
+
+[View tutorial](https://github.com/Code-the-Dream-School/common-instructions/blob/main/common/how-to-merge.md)
+
+Checkout your main branch and pull changes:
+
+    git checkout main
+    git pull
+
+Create a new local branch to work on separate from the `main` branch:
+
+    git checkout -b lesson-1-4
+
+Now, open the project directory in your code editor and continue to the next section.
+
+### Task List:
+
+#### Move Todo List into State
+
+- [ ] Open `/src/App.js`
+- [ ] Create new state variable named `todoList` with setter `setTodoList` and default value of an empty Array
+- [ ] Pass `todoList` state as a prop named `todoList` to the `TodoList` component
```

Comment:
that's a lot of `t/TodoList` haha. Don't think there's much to be done, just found it a bit amusing/potentially confusing? I don't have the context of the students' previous understanding though.

Reply:
yeah that's fair, maybe something we can refine in the future